Ryujinx.Audio.Backends.SDL2/SDL2HardwareDeviceDriver.cs

```diff
@@ -112,13 +113,19 @@ internal static uint OpenStream(SampleFormat requestedSampleFormat, uint request
 
             if (device == 0)
             {
+                string errorMessage = $"SDL2 open audio device initialization failed with error \"{SDL_GetError()}\"";
+
+                Logger.Error?.Print(LogClass.Application, errorMessage);
                 return 0;
             }
 
             bool isValid = got.format == desired.format && got.freq == desired.freq && got.channels == desired.channels;
 
             if (!isValid)
             {
+                string errorMessage = $"SDL2 open audio device is not valid";
+
+                Logger.Error?.Print(LogClass.Application, errorMessage);
```

Comment:
`errorMessage` can be removed (and the string interpolation too).

```suggestion
                Logger.Error?.Print(LogClass.Application, "SDL2 open audio device is not valid");
```